neurodiffeq/generators.py

```diff
@@ -4,21 +4,25 @@
 import numpy as np
 from typing import List
 
+def _chebyshev_uniform(a, b, n):
```

Comment:
I am not sure that this makes sense. Chebyshev nodes are deterministic sequence of numbers. Its not a function which is valid for a random number. 
I think Its better to define "chebyshev_noisy" which adds noise to the chebyshev sequence rather than doing this.

Reply:
Thank you for your comments, Sathvik! I pushed a new commit with an attempt of chebyshev_noisy.